### PR TITLE
Newsletters: add 190 (2022-03-09)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic links @harding 
- [x] PR review club summary @glozow 
- [x] Bitcoin Core 23542 @Xekyo 
- [x] BDK 522 @adamjonas

### CODE REVIEW

File: _posts/en/newsletters/2022-03-09-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+  transaction re-spending those bitcoins or any bitcoins merged with it
+  for perpetuity.  It was [asked][harding recurse] whether anyone
+  had concerns about allowing recursive covenants in Bitcoin, with some
+  of the most notable concerns being summarized below:
+
+    - *Gradual loss of censorship resistance:* contributor Shinobi
+      [posted][shinobi recurse] a repeat of his concerns previously
+      mentioned in [Newsletter #157][news157 csfs] about recursive
+      covenants that gave a powerful third party control over subsequent
+      spending of any coins that entity currently controlled.  For
```
Comment (adamjonas) [inline _posts/en/newsletters/2022-03-09-newsletter.md:31]:
```suggestion
      spending of any coins that the entity currently controlled.  For
```

File: _posts/en/newsletters/2022-03-09-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+      example, a government could require (by law) that its populace
+      only accept coins that the goverment could later seize (as
```
Comment (adamjonas) [inline _posts/en/newsletters/2022-03-09-newsletter.md:33]:
```suggestion
      only accept coins that the government could later seize (as
```

File: _posts/en/newsletters/2022-03-09-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+      authorized to spend a coin had chosen to spend that money.
+      Ideally, any UTXO (coin) can be spent today using a single compact
+      proof that the spend was authorized, such as a 64-byte [schnorr
+      signature][topic schnorr signatures].  Bitcoin already allows more
+      complex scripts to enable the creation of contracts, such as
+      multisignature compacts and protocols like LN, but this capability
+      can be abused to include operations in scripts that aren't
+      necessary to enforce the terms of a contract.  For example,
+      Bitcoin has in the past been at [risk][cve-2013-2292] of
+      denial-of-service attacks from specially-designed tranasctions
```
Comment (adamjonas) [inline _posts/en/newsletters/2022-03-09-newsletter.md:57]:
```suggestion
      denial-of-service attacks from specially-designed transactions
```

File: _posts/en/newsletters/2022-03-09-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+[cve-2013-2292]: /en/topics/cve/#CVE-2013-2292
+[zmn turing]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-February/019928.html
+[zmn drivechains]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-February/019976.html
+[turing completeness]: https://en.wikipedia.org/wiki/Turing_completeness
+[halting problem]: https://en.wikipedia.org/wiki/Halting_problem
+[towns drivechains]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-February/019984.html
+[drivechains vs ln]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-February/019991.html
+[osuntokun bandwidth]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2022-February/003498.html
+[russell reply]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2022-February/003499.html
+[harding recurse]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-February/019885.html
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-03-09-newsletter.md:275]:
```suggestion
[harding recurse]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-February/019885.html
[reviews 23542]: https://bitcoincore.reviews/23542
```

**Comment by harding** [comment]
Made edits for feedback, thanks @adamjonas and @bitschmidty !  Reviewed contributions and made a few small edits, thanks @glozow and @adamjonas !  Added LDK release and topic links.

Thanks everyone!

File: _posts/en/newsletters/2022-03-09-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23542][] removes Bitcoin Core's preference to only
+  connect to peers on the default Bitcoin Port (8333 for mainnet).
```
Comment (glozow) [inline _posts/en/newsletters/2022-03-09-newsletter.md:237]: is Port meant to be capitalized?

**Review by glozow — COMMENTED** [review]
Looks good to me! I just pushed a question mark that was missing from my section

File: _posts/en/newsletters/2022-03-09-newsletter.md (hunk 5 above)
Comment (bitschmidty) [reply _posts/en/newsletters/2022-03-09-newsletter.md:237]: changed!

**Comment by bitschmidty** [comment]
Looks great, team! Thank you for an excellent newsletter @harding @glozow @adamjonas !

**Pull request merged**